src/model/modifier/__tests__/RichTextEditorUtil-test.js

```diff
@@ -82,6 +82,27 @@ describe('RichTextEditorUtil', () => {
       expect(lastBlockNow.getText()).toBe('');
     });
 
+    it('resets the current block type at the start of the first block', () => {
```

Comment:
Thanks for writing a test!

![testing_i_love_testing](https://user-images.githubusercontent.com/1114467/31580240-ba326ae2-b0fe-11e7-8bf5-604687b6472e.jpg)